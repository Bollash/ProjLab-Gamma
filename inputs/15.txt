Create Space
Add Asteroid Iron
Add Settler 0
Add Settler 0
Add Material 0 Iron
Add Material 0 Iron
Add Material 0 Iron
Add Material 0 Coal
Add Material 0 Coal
Add Material 0 Coal
Add Material 1 Ice
Add Material 1 Ice
Add Material 1 Ice
Add Material 1 Uran
Add Material 1 Uran
Add Material 1 Uran
Build Base
Status All