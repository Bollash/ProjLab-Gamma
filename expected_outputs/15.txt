Space létrehozva
A nyersanyaggal teli aszteroida hozzá lett adva a space-hez.
Sikeres volt a telepeshozzáadás
Sikeres volt a telepeshozzáadás
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Sikeresen betettük a nyersanyagot a telepesbe
Felépült a bázis. Nyertek a telepesek
Space
2
10
10
True
1
2
Asteroid 
0
3
3
3
Iron
0
2
0
1
Settler
0
0
0
0
Settler
1
0
0
0